.typos.toml

```diff
@@ -0,0 +1,15 @@
+[default]
+extend-ignore-re = [
+    "Lorem ipsum [a-zA-Z .,]*",
+]
```

Comment:
This regular expression looks like it either:
1) only handles to the end of the first line
2) disables checking for the entire file

I presume that it's option 1, and the perceived typos are only on the first line. But what happens if an update to typos makes something in that second paragraph be treated as a typo?

Reply:
The intention was "until the end of the string literal" (notice how that character class repetition at the end doesn't include quotes), but you might be right that it only matches the first line. Could add `\n` to the character class but I kind of doubt that typos uses multiline regx. If an update to typos detects something in the second paragraph as a typo then (a) we'll only see it when explicitly bumping the version used in CI and (b) I'd say we should try with `\n` then and if it doesn't work, just make a similar regex for the second line, I guess?

Reply:
Oh yeah - my eyes entirely skimmed over the first line

Apologies for the confusion. We definitely should check that it works as expected